composer.json

```diff
@@ -9,7 +9,7 @@
         "composer/installers": "^1.0 || ^2.0"
     },
     "require-dev": {
-        "yoast/phpunit-polyfills": "^1.0"
+        "yoast/phpunit-polyfills": "^1.1.0"
```

Comment:
CI fails: Error: Version mismatch detected for the PHPUnit Polyfills. Please ensure that PHPUnit Polyfills 1.1.0 or higher is loaded. Found version: 1.0.3